### ISSUE TITLE
Refactor the fetch data script

### ISSUE THREAD

**Issue by codeDavidLopez** [amosproj/amos2022ss06-idunion-blockchain-dashboard#43]
## User story
1. As a developer
2. I want / need a parameterized script
3. So that the data can be fetched upon parameter input in command line

## Acceptance criteria
* Indy node monitor adressed not hard coded 
* Command line input available for node monitor adressing

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## Feature DoD 
* Feature has been fully implemented
* All acceptance criteria were met
* All tests are passing
* No known errors
* Documentation updated
* Bill of materials updated

## Sprint Release DoD
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Peer code review passed
* Developers agreed to release
* Documentation updated
* Bill of materials updated

**Comment by A-Sandrini** [comment]
input not in command line but in variables file
